src/MizarWidgets/include/MizarWidgets/MizarMainWindow.h

```diff
@@ -18,8 +18,15 @@ class MainWindow;
 namespace orbit_mizar_widgets {
 
 class MizarMainWindow : public QMainWindow {
+  template <typename T>
```

Comment:
[Declaration order should be public, protected, private](https://google.github.io/styleguide/cppguide.html#Declaration_Order)

Reply:
True. I decided violate that rule here, as I don't see a better option. We need these clauses on top, otherwise the public and protected section will not see them. Is there a better workaround?

Reply:
Hm, I'd just move `public:` up to make the using declarations public as well. You don't need them in public but it doesn't hurt either.

Reply:
But the client code might rely on these aliases and I might wish to remove them later.